docu/files/LocalizedEditor.html.md

```diff
@@ -12,7 +12,9 @@ The localized editor is a custom Angular directive in **builder_editors** with a
 
 As a Builder module developer, you need to complete a few simple tasks:
```

Comment:
I know this sentence wasn't updated for this pull request, but can we remove "you need to" and "if you want to" language while we're in here?